apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog/GoogleStackdriverTarget_Configuration.cs

```diff
@@ -114,6 +114,13 @@ public partial class GoogleStackdriverTarget
         /// </remarks>
         public Layout SpanId { get; set; }
 
+        /// <summary>
+        /// True means that the trace resource name in the trace field was sampled for storage in a trace backend.
+        /// False means that the trace was not sampled for storage when this log entry was written, or the sampling decision was unknown at the time.
+        /// A non-sampled trace value is still useful as a request correlation identifier. The default is False.
+        /// </summary>
+        public bool TraceSampled { get; set; }
```

Comment:
This value could be different per trace, it needs to be extracted from the log entry, it's not a configuration value.

Reply:
Changed to NLog Layout that accepts "true" or "1".